config/rbac/nifiresource_editor_role.yaml

```diff
@@ -0,0 +1,27 @@
+# permissions for end users to edit nifiresources.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/name: nifikop
+    app.kubernetes.io/managed-by: kustomize
+  name: nifiresource-editor-role
+rules:
+- apiGroups:
+  - nifi.konpyutaika.com
+  resources:
+  - nifiresources
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - nifi.konpyutaika.com
+  resources:
+  - nifiresources/status
+  verbs:
+  - get
```

Comment:
Could look into the other corresponding files for the other resources and match the content (like for the label). Not sure why they are different, `operator-sdk` changed since the beginning, but just to be sure.